ota-plus-web/app/reactapp/src/components/campaign/InstallationReportView.jsx

```diff
@@ -78,7 +78,12 @@ class InstallationReportView extends Component {
                 </div>
                 {alphaPlusEnabled && (
                   <div className='col-actions'>
-                    <div className='failure_report' onClick={() => {showRetryModal(failure.name)}}>
+                    <div
```

Comment:
Let's avoid formatting/prettying/whitespace changes in `feat/` and `bugfix/` branches. Do this in separate `refactor/` branches.